integration-tests/maven/build.gradle.kts

```diff
@@ -0,0 +1,17 @@
+import org.jetbrains.dependsOnMavenLocalPublication
+
+evaluationDependsOn(":runners:maven-plugin")
+
+dependencies {
+    implementation(kotlin("stdlib"))
+    implementation(kotlin("test-junit"))
+}
+
+tasks.integrationTest {
+    dependsOnMavenLocalPublication()
+    dependsOn(":runners:maven-plugin:setupMaven")
+
+    val dokka_version: String by project
+    environment("DOKKA_VERSION", dokka_version)
+    environment("MVN_BINARY_PATH", project(":runners:maven-plugin").extra["MVN_BINARY_PATH"].toString())
```

Comment:
@kamildoleglo Any ideas? 😨

Reply:
I will create a proper task called "setupMaven" that will hopefully make this better 👍